docs/volume_annotation.md

```diff
@@ -34,40 +44,30 @@ The "merger mode" is available in skeleton and hybrid annotation mode. Mark conn
 
 "Merger mode" can be enabled in the settings under "Nodes & Trees" with the option "Enable Merger Mode". As soon as you enable it, all already existing trees will be used to form merged segments.
 
+To fix split errors see section on flood filling below.
+
 ![Video: Proof-Reading Volume Annotations](https://www.youtube.com/watch?v=Sq4AuWanK14)
 
+### Volume Flood Fills
+webKnossos supports volumetric flood fills (3D) to relabel a segment with a new ID. Instead of having the relabela segment slice-by-slice webKnossos can do this for you. This operation allows you to fix both split and merge errors:
```

Comment:
```suggestion
webKnossos supports volumetric flood fills (3D) to relabel a segment with a new ID. Instead of having the relabel segment slice-by-slice, webKnossos can do this for you. This operation allows you to fix both split and merge errors:
```